### DISCUSSION TITLE
Get ESP-01S working with a BTT TFT 35 E3 V3.0 when running version 3 of ESP3D

### DISCUSSION THREAD

**Discussion by TheZoker** [luc-github/ESP3D#984] [Questions]
Hi there,

I'm using a BTT ESP-01S with the version 3 of the ESP3D firmware. The ESP-01S is plugged into a BTT TFT 35 E3 V3.0 screen.
I was able to upload the index.html.gz file and set the correct wifi. The chip is connected to my wifi and I can access the web UI. But for some reason, I don't get any data from the printer. I set the baudrate to 115200. I also tried to change the baudrate within the UI, but I always get a "Error saving data" message.

Is this the correct baudrate? Do I need to change something within the config of the TFT to get this working? 

![image](https://github.com/luc-github/ESP3D/assets/1368405/67ed811c-28f1-4f48-ae72-f61f1aca59b1)

![image](https://github.com/luc-github/ESP3D/assets/1368405/1d546541-6fe3-4c31-8267-63d63de0914a)

Thanks!

**Comment by luc-github** [comment]
Did you configured your tft ini file and put on TFT SD to communicate with ESP3D ?
https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/blob/master/Copy%20to%20SD%20Card%20root%20directory%20to%20update/config.ini
